pandas-stubs/core/frame.pyi

```diff
@@ -100,6 +103,7 @@ from pandas._typing import (
     MergeHow,
     NaPosition,
     NDFrameT,
+    P,
```

Comment:
Yeah no problem!

Reply:
> Also, what's the rationale here (to be clear—not a pushback, just honestly curious)? I see both are used throughout the codebase.

Since we did an `import T as Ttype`, I figured it would be better to be consistent.

Reply:
These tests have now been implimented!

Reply:
> So this change has been undone—like you suggested—and moved to `generic.pyi` instead. There `TypeVar` is just specified as `T`—not `TType`—so I left `ParamSpec` as `P`—not `PType`. Happy to move to either in the end.

That's fine.